pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2490,13 +2490,36 @@ cdef class SemiMonthBegin(SemiMonthOffset):
 
 cdef class Week(SingleConstructorOffset):
     """
-    Weekly offset.
+        Weekly offset.
 
-    Parameters
-    ----------
-    weekday : int or None, default None
-        Always generate specific day of week. 0 for Monday.
-    """
+        Parameters
+        ----------
+        weekday : int or None, default None
+            Always generate specific day of week. 0 for Monday and 6 for Sunday.
+
+        Examples
+        ---------
+        ## Below examples explain the usage of this object.
+        ## Importing modules
+        import pandas as pd
+
+        date_format = "%Y-%m-%d"
+        date_object = pd.to_datetime("2023-01-13",format = date_format)
+        print(date_object)
+        ## 2023-01-13 00:00:00
```

Comment:
this isn't how we write examples, please check https://pandas.pydata.org/docs/development/contributing_docstring.html